doc/api/events.md

```diff
@@ -1266,6 +1266,57 @@ const { getEventListeners, EventEmitter } = require('node:events');
 }
 ```
 
+## `events.getMaxListeners(emitterOrTarget)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `emitterOrTarget` {EventEmitter|EventTarget}
+* Returns: {number}
+
+Returns the currently set max amount of listeners.
+
+For `EventEmitter`s this behaves exactly the same as calling `.getMaxListeners` on
+the emitter.
+
+For `EventTarget`s this is the only way to get the max event listeners for the
```

Comment:
What's "the max event listeners" on an EventTarget?

Reply:
Ah, I didn't even remember 'kMaxEventTargetListeners". Please add some info to the docs (that beyond this number Node.js warns etc)?